onnxruntime/core/providers/cpu/tensor/eye_like.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+
+namespace onnxruntime {
+
+class EyeLike final : public OpKernel {
+ public:
+  EyeLike(const OpKernelInfo& info) : OpKernel(info) {
+    if (!info.GetAttr("k", &k_).IsOK()) {
+      ONNXRUNTIME_THROW("Missing 'k' attribute value");
```

Comment:
Looks like 'k' is optional based on the spec in which case we don't have to throw.